File: replace_colors.js
const fs = require('fs');
const path = require('path');

const files = [
    "app/globals.css",
    "components/logo.tsx",
    "components/navbar.tsx",
    "components/landing/reviews.tsx",
    "components/landing/hero.tsx",
    "components/landing/footer.tsx",
    "components/landing/features.tsx",
    "components/landing/cta.tsx",
    "app/auth/sign-up/page.tsx",
    "app/auth/login/page.tsx",
    "app/(dashboard)/teacher-dashboard/page.tsx"
];

const basePath = "e:/Projects/ShikkhaGriho/v0-ShikkhaGriho/";

for (const relPath of files) {
    const fullPath = path.join(basePath, relPath);
    if (fs.existsSync(fullPath)) {
        let content = fs.readFileSync(fullPath, 'utf8');
        const origContent = content;

        content = content.replace(/1E3A8A/gi, '3F7D4E');
        content = content.replace(/1D4ED8/gi, '2F6B3D');
        content = content.replace(/EFF6FF/gi, 'E6F4EA');

        if (content !== origContent) {
            fs.writeFileSync(fullPath, content, 'utf8');
            console.log(`Updated ${fullPath}`);
        }
    } else {
        console.log(`File not found: ${fullPath}`);
    }
}

File: replace_colors.py
import os
import re

# Files to update based on grep search results
files = [
    r"app/globals.css",
    r"components/logo.tsx",
    r"components/navbar.tsx",
    r"components/landing/reviews.tsx",
    r"components/landing/hero.tsx",
    r"components/landing/footer.tsx",
    r"components/landing/features.tsx",
    r"components/landing/cta.tsx",
    r"app/auth/sign-up/page.tsx",
    r"app/auth/login/page.tsx",
    r"app/(dashboard)/teacher-dashboard/page.tsx"
]

replacements = {
    re.compile(r'1E3A8A', re.IGNORECASE): '3F7D4E',
    re.compile(r'1D4ED8', re.IGNORECASE): '2F6B3D',
    re.compile(r'EFF6FF', re.IGNORECASE): 'E6F4EA' 
}

for rel_path in files:
    filepath = os.path.join(r"e:/Projects/ShikkhaGriho/v0-ShikkhaGriho/", rel_path)
    if os.path.exists(filepath):
        with open(filepath, 'r', encoding='utf-8') as f:
            content = f.read()
        
        original_content = content
        for pattern, replacement in replacements.items():
            content = pattern.sub(replacement, content)
            
        if content != original_content:
            with open(filepath, 'w', encoding='utf-8') as f:
                f.write(content)
            print(f"Updated {filepath}")
    else:
        print(f"File not found: {filepath}")
